Framework/Algorithms/src/UnGroupWorkspace.cpp

```diff
@@ -53,6 +52,11 @@ void UnGroupWorkspace::exec() {
   if (!wsGrpSptr) {
     throw std::runtime_error("Selected Workspace is not a WorkspaceGroup");
   }
+  // add algorithm history to the workspaces being released from the group
+  WorkspaceVector outputWorkspaces;
+  for (const auto &ws : wsGrpSptr->getAllItems())
+    outputWorkspaces.emplace_back(ws);
+  fillHistory(outputWorkspaces);
```

Comment:
Is it possible to just do this?
```suggestion
  fillHistory(wsGrpSptr->getAllItems());
```

Reply:
yes 🙃 clearly did that when tired